StepLang.Tooling.Formatting/StringExtensions.cs

```diff
@@ -4,6 +4,11 @@ namespace StepLang.Tooling.Formatting;
 
 public static class StringExtensions
 {
+    public static IEnumerable<string> SplitLines(this string input)
+    {
+        return input.Split("\r\n").SelectMany(l => l.Split("\n"));
```

Comment:
does the `\r` go away? if so, maybe `NormalizeLinebreaks`?

Reply:
See other comment on `SplitLines`